CONTRIBUTING.md

```diff
@@ -1,14 +1,50 @@
 # Contributing to GWpy
 
-This is [@duncanmmacleod](//github.com/duncanmmacleod/)'s workflow, which might work well for others, it is just a verbose version of the [GitHub flow](https://guides.github.com/introduction/flow/).
+## Reporting Issues
+
+When opening an issue to report a problem, please try to provide a minimal code
+example that reproduces the issue along with details of the operating
+system and the Python, NumPy, Astropy, and GWpy versions you are using.
+
+## Contributing Code
+
+**Imposter syndrome disclaimer**: We want your help. No, really.
+
+There may be a little voice inside your head that is telling you that you're not
+ready to be an open source contributor; that your skills aren't nearly good
+enough to contribute. What could you possibly offer a project like this one?
+
+We assure you - the little voice in your head is wrong. If you can write code at
+all, you can contribute code to open source. Contributing to open source
+projects is a fantastic way to advance one's coding skills. Writing perfect code
+isn't the measure of a good developer (that would disqualify all of us!); it's
+trying to create something, making mistakes, and learning from those
+mistakes. That's how we all improve, and we are happy to help others learn.
+
+Being an open source contributor doesn't just mean writing code, either. You can
+help out by writing documentation, tests, or even giving feedback about the
+project (and yes - that includes giving feedback about the contribution
+process). Some of these contributions may be the most valuable to the project as
+a whole, because you're coming to the project with fresh eyes, so you can see
+the errors and assumptions that seasoned contributors have glossed over.
+
+Note: This disclaimer was originally written by
+[Adrienne Lowe](https://github.com/adriennefriend) for a
+[PyCon talk](https://www.youtube.com/watch?v=6Uj746j9Heo), and was adapted by
+GWpy based on its use in the [Astropy](https://github.com/astropy/astropy/)
+contributing guide.
+
+## Development model
+
+Contributions to GWpy are made via pull requests from GitHub users' forks of the main [gwpy repositories](https://github.com/gwpy/gwpy), following the [GitHub flow](https://guides.github.com/introduction/flow/) workflow.
```

Comment:
@duncanmmacleod, did you mean to point to the stable mainline development model here?

Reply:
No, the included links is easier to follow. While stable mainline is the actual model, that's mainline for the maintainers (specifically, the release managers) to help organise things.